docs/New-Maintainer-Checklist.md

```diff
@@ -55,7 +55,7 @@ If they accept, follow a few steps to get them set up:
 - Add them to the [Jenkins' GitHub Authorization Settings admin user names](https://jenkins.brew.sh/configureSecurity/) so they can adjust settings and restart jobs
 - Add them to the [Jenkins' GitHub Pull Request Builder admin list](https://jenkins.brew.sh/configure) to enable `@BrewTestBot test this please` for them
 - Invite them to the [`homebrew-maintainers` private maintainers mailing list](https://lists.sfconservancy.org/mailman/admin/homebrew-maintainers/members/add)
-- Invite them to the [`machomebrew` private maintainers Slack](https://machomebrew.slack.com/admin/invites)
+- Invite them to the [`machomebrew` private maintainers Slack](https://machomebrew.slack.com/admin/invites) (and ensure they've read the [communication guidelines](Maintainer-Guidelines.md#communication))
```

Comment:
should that be #Communication not #communication ?

Reply:
@ilovezfs from my testing `#communication` is correct/works.

Reply:
ok